backend/engine/plugins/trivy_sca/main.py

```diff
@@ -0,0 +1,66 @@
+"""
+trivy SCA plugin
+"""
+import json
+import subprocess
+from engine.plugins.lib import utils
+from engine.plugins.lib.trivy_common.generate_locks import check_package_files
+from engine.plugins.lib.trivy_common.parsing_util import convert_output
+from engine.plugins.lib.trivy_common.parsing_util import parse_output
+
+logger = utils.setup_logging("trivy_sca")
+
+NO_RESULTS_TEXT = "no supported file was detected"
+
+
+def execute_trivy_lock_scan(path: str, include_dev: bool):
+    # passing in "include dev" tag if include_dev arg is True
+    logger.info(f"Scanning lock-files. Dev-dependencies: {include_dev}")
+    if include_dev:
+        proc = subprocess.run(
+            ["trivy", "fs", "--include-dev-deps", path, "--format", "json"], capture_output=True, check=False
```

Comment:
should we define the list of args once, and just remove `--include-dev-deps` if `include_dev` is `False`?

e.g.:
```
args = ["trivy", "fs", "--include-dev-deps", path, "--format", "json"]
if not include_dev:
  args.remove("--include-dev-deps")
proc = subprocess.run(args, capture_output=True, check=False)

Reply:
also, same question here as previous comment, is there output we want that `--quiet` would omit?

Reply:
Yea I think that would be a lot cleaner. Thanks for that suggestion, will add that in!